web_ir_actions_act_multi/models/ir_actions.py

```diff
@@ -0,0 +1,15 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class IrActionsActMulti(models.Model):
+    _name = "ir.actions.act_multi"
+    _description = "Action Mulit"
+    _inherit = "ir.actions.actions"
+    _table = "ir_actions"
```

Comment:
This is pretty dangerous without an uninstall hook as seen in https://github.com/OCA/web/issues/2394

Reply:
wow... thats LGTM... i will implement that fix